electron_app/src/DBManager.js

```diff
@@ -1080,12 +1080,13 @@ const getAllFeedItems = () => {
   return db.select('*').from(Table.FEEDITEM);
 };
 
-const updateFeedItem = ({ id, seen }) => {
+const updateFeedItem = ({ ids, id, seen }) => {
```

Comment:
Can you handle only one identifier?
`const updateFeedItem = ({ ids, seen }) => ... `